.changeset/bright-papayas-speak.md

```diff
@@ -0,0 +1,7 @@
+---
+"@finos/legend-art": minor
+"@finos/legend-extension-external-language-morphir": minor
+"@finos/legend-graph": minor
+"@finos/legend-studio": minor
```

Comment:
Add a separate changeset for `@finos/legend-studio`.
Add a separate changeset for `@finos/legend-extension-external-language-morphir`

Reply:
Created separate changesets.